db/migrate/20190305150816_drop_elements_from_vm_attrs_hash.rb

```diff
@@ -0,0 +1,68 @@
+class DropElementsFromVmAttrsHash < ActiveRecord::Migration[5.1]
+  def up
+    say "Formatting nics attributes, total #{FakeNic.unscoped.count}"
```

Comment:
formatting sounds scary, you can say something like "correcting serialized nics attributes...", and the count should be just the affected ones, not all of them (see comment below for getting just that)